lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -83,10 +83,15 @@ def __init__(
 
         self.diffusion = DiffusionModel(config)
 
+        image_keys = [k for k in config.input_shapes if k.startswith("observation.image")]
+        # Note: This check is covered in the post-init of the config but have a sanity check just in case.
+        assert len(image_keys) == 1
+        self.input_image_key = image_keys[0]
```

Comment:
```suggestion
        image_keys = [k for k in config.input_shapes if k.startswith("observation.image")]
        # Note: This check is covered in the post-init of the config but have a sanity check just in case.
        # TODO(alexander-soare): make diffusion compatible with multiple image keys
        assert len(image_keys) == 1
        self.input_image_key = image_keys[0]
```

Reply:
Added in main docstring.